### PR TITLE
Show a warning in the classic ui when plone.volto is installed

### PR DESCRIPTION
**Pull request by pbauer**
Fixes https://github.com/plone/Products.CMFPlone/issues/3664

### CODE REVIEW

**Comment by pbauer** [comment]
@jenkins-plone-org please run jobs

File: src/plone/volto/browser/voltobackendwarning.pt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<tal:statusmsg tal:define="icons nocall: context/@@iconresolver"
+               i18n:domain="plone">
+
+    <div class="portalMessage statusmessage statusmessage-warning alert alert-warning"
+         role="alert">
+        <tal:icon tal:replace="structure python:icons.tag('plone-statusmessage-warning', tag_alt='warning', tag_class='statusmessage-icon mb-1 me-2')" />
+        <strong i18n:translate="">Warning</strong>:
+        <span class="content">
+            <tal:warning i18n:translate="volto_backend_warning">This site is configured to use the Volto frontend. Your are accessing the backend.</tal:warning>
```
Comment (davisagli) [inline src/plone/volto/browser/voltobackendwarning.pt:9]:
```suggestion
            <tal:warning i18n:translate="volto_backend_warning">This site is configured to use the Volto frontend. You are accessing the backend.</tal:warning>
```

File: src/plone/volto/browser/voltobackendwarning.pt [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            <br />
+            <tal:warning i18n:translate="volto_backend_warning_link">For more information please read the <a href="https://6.dev-docs.plone.org/volto/index.html">documentation</a></tal:warning>
```
Comment (davisagli) [inline src/plone/volto/browser/voltobackendwarning.pt:11]:
```suggestion
            <tal:warning i18n:translate="volto_backend_warning_link">For more information, please read the <a href="https://6.dev-docs.plone.org/volto/index.html">documentation</a>.</tal:warning>
```

File: news/94.bugfix [hunk 3]
```diff
@@ -0,0 +1 @@
+Show a warning in the classic ui when plone.volto is installed. Fix https://github.com/plone/Products.CMFPlone/issues/3664 [pbauer]
```
Comment (davisagli) [inline news/94.bugfix:1]:
```suggestion
Show a warning in the classic UI when plone.volto is installed. Fix https://github.com/plone/Products.CMFPlone/issues/3664 [pbauer]
```

**Review by davisagli — APPROVED** [review]
@stevepiercy Does this look all right for where to link in the docs?

**Comment by davisagli** [comment]
@jenkins-plone-org please run jobs

**Comment by stevepiercy** [comment]
> @stevepiercy Does this look all right for where to link in the docs?

I'm not sure. If the intent is to direct the user to read all the documentation on Volto, then yes. However from the original issue, I think the user is attempting to edit content, not learn all about Volto. I would suggest that a better link would be to edit content in the frontend itself (http://localhost:3000/ or whatever it is running on), not a link to the documentation of Volto.

**Comment by davisagli** [comment]
@stevepiercy I think the goal here is not about understanding how to edit, but more to help the admin who is creating the Plone site understand that they need to also set up Volto and access the site at Volto's URL.

I see your point that the most helpful thing would be to link to the Volto URL, but the backend has no way of knowing what that is if it wasn't accessed through Volto to begin with.

**Comment by stevepiercy** [comment]
In that case, I think a better link is to the docs on Installation which is more to the point.

https://6.dev-docs.plone.org/install/install-from-packages.html

**Comment by pbauer** [comment]
It seems we are still lacking good docs on "There are two frontends, how do I choose the right one and what do I set it up?"

https://training.plone.org/mastering-plone/what_is_plone.html#classic-frontend has a bit of that but we need that at the beginning of the installation docs.

**Comment by stevepiercy** [comment]
First, I think this issue is that the user did not start up the backend and had already made a choice of frontend. @mauritsvanrees can you provide more insight?

Second, I acknowledge that we need to give direction to users for what frontend to choose in the Plone 6 Documentation. I think a new issue in `plone/documentation` to help users make a choice of frontends would address that matter. @pbauer's do you agree?

We laid out a skeleton in [Overview](https://6.dev-docs.plone.org/overview/index.html) for a comprehensive description of features. That would be one place for it.

Another good spot to make a decision between frontends would be in [Install](https://6.dev-docs.plone.org/install/index.html). But it would need some careful thought, not just a copy-pasta, because we already present two decisions already, "demo or install?" and "which install method?". I would be OK to move the demo stuff above the "demo or install" decision tree to make it less complicated. I have to try out a few layouts.

**Comment by mauritsvanrees** [comment]
@stevepiercy No, the issue fixed here is that the user has started up the backend and created a default Plone site. This means a site that expects to be run with a Volto frontend. The user does not know that he should start a frontend, and is left with a site that only partially works. With this PR, we tell the user what to do.

**Comment by stevepiercy** [comment]
Let me make sure I understand the story, after re-reading everything.

1. The user already decided to use Volto for a frontend.
2. They installed Plone backend
3. They installed Volto for frontend.
4. They started up the Plone backend (which also means they started Classic UI).
5. They did not start Volto frontend.
6. They visited Classic UI, perhaps thinking that it was Volto.

@mauritsvanrees is that correct?

**Comment by mauritsvanrees** [comment]
@stevepiercy No. In this use case the users has no clue that Volto even exists. So from your list we only have step 2 and 4, and a bit of 6:

2. They installed Plone backend.
4. They started up the Plone backend (which also means they started Classic UI).
6. They visited Classic UI, and it just does not quite work.

**Comment by stevepiercy** [comment]
OK, one more try. Is it correct that the user in this case needs to:

1. Install Volto, if not already installed.
2. Start Volto, if not already started.
3. Visit the Volto frontend.

If that is correct, then I think the best page in the docs is https://6.dev-docs.plone.org/install/install-from-packages.html instead of the current https://6.dev-docs.plone.org/volto/index.html.

**Comment by mauritsvanrees** [comment]
@stevepiercy Yes, now you have it.
I think both links can work. The current one quickly leads you to the Getting Started section, which works. Install from packages is a nice one too. Fine if someone wants to change this.

**Pull request merged**